dockerfile_parse/parser.py

```diff
@@ -236,3 +264,73 @@ def labels(self):
                         labels[key_val[0]] = key_val[1] if len(key_val) > 1 else ''
                         logger.debug("new label %s=%s", repr(key_val[0]), repr(labels[key_val[0]]))
         return labels
+
+    def modify_instruction_label(self, label_key, label_value):
```

Comment:
I guess a more Pythonic way of doing it would be to use setters for the properties, but would that be really hard?